draft-ietf-regext-rdap-geofeed.md

```diff
@@ -32,107 +32,208 @@ email = "tomh@apnic.net"
-document are to be interpreted as described in BCP 14 [@!RFC2119] [@!RFC8174]
-when, and only when, they appear in all capitals, as shown here.
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD","SHOULD NOT", "RECOMMENDED",
+"NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14
+[@!RFC2119] [@!RFC8174] when, and only when, they appear in all capitals, as shown here.
 
 # Specification
 
 ## Extension
 
-A new RDAP extension `geofeedv1` is defined for accessing the geofeed data
-through RDAP. It extends the IP Network object class [@!RFC9083] to include
-a new `geofeed` member.
+A new RDAP extension "geofeed1" is defined for accessing the geofeed data through RDAP. It updates the IP Network object
+class definition ([@!RFC9083, section 5.4]) to include a new link object for the geofeed file URL in its "links" array
+([@!RFC9083, section 4.2]).
 
-An RDAP server conforming to this specification MUST include the `geofeedv1`
-extension string in the `rdapConformance` array for the IP Network lookup and
-search responses, as well as in the help response.
+An RDAP server conforming to this specification MUST include the "geofeed1" extension string in the "rdapConformance"
+array for the IP Network lookup and search responses, as well as in the help response.
 
-## Geofeed Member
+## Geofeed Link
 
-The IP Network object class is extended to include a new `geofeed` member, as
-specified below:
+The IP Network object class ([@!RFC9083, section 5.4]) MAY include a link object for the geofeed file URL in its "links"
+array, with the following REQUIRED JSON members:
 
-geofeed — a string specifying the HTTPS URL of the geofeed file for an IP
-          network [@!RFC9092]
+* "value" -- The "value" JSON value is the context URI and set to an IP Network lookup URL ([@!RFC9082, section 3.1.1]).
+* "rel" -- The "rel" JSON value is the link relation type and set to the "geofeed" string. The "geofeed" link relation
+type is new and will be registered in the IANA Link Relations Registry (see the "Link Relations Registry" section).
+* "href" -- The "href" JSON value is the target URI and set to the HTTPS URL of the geofeed file for the IP network in
+the context URI.
 
-Since this specification extends the base definition of the IP Network object
-class, per [@!RFC7480, section 6], the `geofeed` member name MUST be prefixed
-with the `geofeedv1` extension string, followed by an underscore:
-`geofeedv1_geofeed`.
+Per the definition of a web link ([@!RFC8288]), a "geofeed" link object MAY have additional OPTIONAL JSON members.
+Specifically:
 
-The `geofeed` member is OPTIONAL for the `geofeedv1` extension. Furthermore,
-the Redaction by Removal method [@?I-D.ietf-regext-rdap-redacted] SHOULD be
-used when redacting this member.
+* "type" -- The "type" JSON value is the media type for the target URI. See the "Media Type for a Geofeed Link" section
+for acceptable values.
```

Comment:
Since we don't know how the media types for the geofeed file evolve in the future, keeping the "type" member optional. On the other side, we could make it REQUIRED with "application/geofeed+csv" value for a start. Thoughts?

Reply:
Agreed, especially since we are proceeding with "geo" rel.